src/js/pre-logic.js

```diff
@@ -113,13 +182,13 @@ function crunchSprite(parentState) {
     if (isAnEdge) {
       switch(direction) {
       case 'left':
-        return {...state, x: parentState.board.rows};
+        return {...state, x: parentState.board.rows - 1};
```

Comment:
the reason it's a higher order function is to bind the state and then call that function on each individual sprite without having to specify `state.pacman.x`, `state.orange.x` or `state.red.x` , so i think it needs to be bound, right? unless I'm completely misunderstanding what you mean